---
layout: default
title: Assignment File
parent: Input Files
nav_order: 1
---

INPUT FILES
=======

`PHRAPL` requires two input files: an **assignment file** (e.g. cladeAssignments.txt) and a file with **phylogenetic trees in newik** format (e.g. trees.tre). The file of trees must include one tree per gene or independent marker. __<font color='#ff7700'>All files must have the same taxon names (labels) across all loci.</font>__

**You can have missing data.** `PHRAPL` does not require each gene tree to include all individuals/alleles per population/species, but need to include at least 2 or 3 preferably.

**You can include both haploid and diploid data the same analyses** using the argument `popScaling` in the `GridSearch`function.
---
# Assignment file
The assignment file (cladeAssignments.txt) specifies which individuals belong to which populations/species/groups. 

```markdown
## Example of an input assignment file. 
# The first column lists individuals 
# The second column lists populations.
# This example does not include an outgroup.
Indv  PopLabel
ind1	A
ind2	A
ind3	A
ind4	B
ind5	B
ind6	B
ind7	C
ind8	C
ind9	C
ind10	D
ind11	D
ind12	D
```

If an outgroup is included in the analysis, it must be listed at the end.
```markdown
## Example of input assignment file with an outgroup. 
# The first column lists individuals 
# The second column lists populations.
Indv	PopLabel
ind1	A
ind2	A
ind3	A
ind4	B
ind5	B
ind6	B
ind7	C
ind8	C
ind9	C
ind10	D
ind11	D
ind12	D
ind13	outgroup
```
