---
layout: default
title: Gene Trees
parent: Input Files
nav_order: 2
---

INPUT FILES
=======

`PHRAPL` requires two input files: an **assignment file** (e.g. cladeAssignments.txt) and a file with **phylogenetic trees in newik** format (e.g. trees.tre). The file of trees must include one tree per gene or independent marker. __<font color='#ff7700'>All files must have the same taxon names (labels) across all loci.</font>__

**You can have missing data.** `PHRAPL` does not require each gene tree to include all individuals/alleles per population/species, but need to include at least 2 or 3 preferably.

**You can include both haploid and diploid data the same analyses** using the argument `popScaling` in the `GridSearch`function.
---
# Gene trees
`PHRAPL` raeds one file that must include gene trees (locus1_tree; locus2_tree; locus3_tree; locusn_tree;) in **newick** format

```markdown
## Example of file with phylogenetic trees (3 loci)
((ind6,(ind4,ind5),ind11),(ind3,ind9),(((ind12,(ind10,ind7)),ind8),(ind1,ind2)));
(((((ind10,ind7,ind1),ind12),ind6),((ind11,ind2,ind9),ind8)),(ind3,(ind4,ind5)));
(((ind10,ind11),(((ind1,ind12,ind3),(ind2,ind4,ind8,(ind5,ind6))),ind9)),ind7);
```


## Merge all trees into one file
If the tree files are separated, there are two options to merge all the trees in one file:
*	In the shell (in your working directory): 
```markdown
cat *.tre >> trees.tre
```

*	In `PHRAPL` use the function `MergeTrees`:  
```markdown
library(phrapl)
MergeTrees(treePath="/path_to_tree_files/")
```
