---
layout: default
title: NMNH-workshop
nav_order: 8
has_children: true
has_toc: false
permalink: Content/2021-NMNH-workshop
---
**Materials are being updated (will be available on Wednesday August 18)**

# **Smithsonian Species Delimitation Workshop**
August 16-20, 2021 – **<font color='#006579'>PHRAPL</font> workshop held on August 18 from 1:00 pm to 2:30 pm (GMT-4).**
[Zoom link](https://zoom.us/j/95658176247?pwd=ekl0eHFrQnZVU3FtVFBmUE1zamVnZz09) (you may need to pre-register).
[Join the Slack channel.](https://sppdelimitworkshop.slack.com/archives/C028GRU9N01)

---
## Installation
**<font color='#ff7700'>Please, install `PHRAPL` before the tutorial session begins.</font>**

– [Conda Intro / Install](https://github.com/SmithsonianWorkshops/Species_Delimitation_2021_08/blob/main/conda_instructions.md)

– [Create conda environment and install `PHRAPL`](https://phrapl.github.io/Content/1.Installation.html) 

---
# Tutorials

## **[Tutorial 1:](https://phrapl.github.io/Content/2021-NMNH-workshop/NMNH.aa.PHRAPL.tutorial1.html)** The basis of demographic model selection with PHRAPL 
Simple example to get familiar with the basics of `PHRAPL` and analyze a simulated dataset of 3 populations. <br/>
Reference papers:
* Jackson N, Morales AE, Carstens BC, O'Meara BC (2017) [PHRAPL: Phylogeographic Inference using Approximate likelihoods](https://academic.oup.com/sysbio/article/66/6/1045/2999288). Systematic Biology. 66:1045-1053. <br/>
* Carstens BC, Morales AE, Jackson N, O’Meara BC (2017) [Objective choice of Phylogeographic Models](https://www.sciencedirect.com/science/article/pii/S1055790317303160?via%3Dihub). Molecular Phylogenetics and Evolution. 116:136-140. <br/>


## **[Tutorial 2:](https://phrapl.github.io/Content/2021-NMNH-workshop/NMNH.ab.PHRAPL.tutorial2.html)** Divergence with gene flow vs. secondary contact

* **Part 1:** Perform sensitivity analyses in subsets of loci. The analysis includes haploid and diploid data. Dataset of 2 populations of *Plethodon* salamanders. <br/>
> ** – Do I need more data? ** This analysis can help you answer that question.
* **Part 2:** Compare models with gene flow that occurred at different time intervals. Dataset of 2 unisexual and sexual species of *Ambystoma* salamanders. <br/>

Recommended reads of empirical examples:
* Morales AE, Jackson N, Dewey T, O’Meara BC, Carstens BC (2017) [Speciation with gene flow in North American *Myotis* bats](https://academic.oup.com/sysbio/article/66/3/440/2682289). Systematic Biology. 66:440-452. <br/>
* Denton RD, Morales AE, Gibbs HL (2018) [Genome-specific histories of divergence and introgression between an allopolyploid unisexual salamander lineage and two ancestral sexual species](https://onlinelibrary.wiley.com/doi/full/10.1111/evo.13528). Evolution.  72:1689–1700. <br/>
* Mays H, Oehler D, Morrison K, Morales A, Lycans A, et al., Weakley L (2019) [Phylogeography, population structure, gene flow and species delimitation in Rockhopper penguins (*Eudyptes chrysocome* and *Eudyptes moseleyi*)](https://academic.oup.com/jhered/article/110/7/801/5583958?login=true). Journal of Heredity. esz051, doi.org/10.1093/jhered/esz051. <br/>

## **[Tutorial 3:]()** Species delimitation
Example for how to test the existance of 1, 2, or 3 species using the first three models from Tutorial 1. <br/>

Reference papers:
* Jackson N, Carstens BC, Morales AE, O’Meara BC (2017) [Species delimitation with gene flow](https://academic.oup.com/sysbio/article/66/5/799/2726792?searchresult=1). Systematic Biology. 66:799-812.


---
## Workshop Info

– [Agenda](https://docs.google.com/document/d/1onnx5ixblgBtEk8akRsvFmvtLPl3-8xl5Hoq6JUhb0g/edit)

– [Workshop GitHub - materials for other packages](https://github.com/SmithsonianWorkshops/Species_Delimitation_2021_08)

Contact Rebeca Dikow <dikowr@si.edu> or Andrea Quattrini <QuattriniA@si.edu> if you want to access recored materials after the workshop.